cmd/client.go

```diff
@@ -303,6 +303,55 @@ func (cli *Client) ShowBridge(c *clipkg.Context) error {
 	return cli.renderResponse(resp, &bridge)
 }
 
+// RemoveBridge removes a specific Bridge by name.
+func (cli *Client) RemoveBridge(c *clipkg.Context) error {
+	cfg := cli.Config
+	if !c.Args().Present() {
+		return cli.errorOut(errors.New("Must pass the name of the bridge to be removed"))
+	}
+	resp, err := utils.BasicAuthDelete(
+		cfg.BasicAuthUsername,
+		cfg.BasicAuthPassword,
+		cfg.ClientNodeURL+"/v2/bridge_types/"+c.Args().First(),
+		"application/json",
+		nil,
+	)
+	if err != nil {
+		return cli.errorOut(err)
+	}
+	defer resp.Body.Close()
+	var bridge models.BridgeType
+	return cli.renderResponse(resp, &bridge)
+}
+
+// RemoveBridges removes several bridges based on JSON specifications
+func (cli *Client) RemoveBridges(c *clipkg.Context) error {
```

Comment:
How useful is this? DELETE on multiple resources introduces some complications we need to consider.

Reply:
I would say the use cases are limited for `Bridges` and it's more of a convenience thing. (I personally found it useful to clean up all the test bridges I had added on a node in one go. It could also come handy if s/b has a bunch of useless/out of service bridges that they want to cleanup). 

However being able to DELETE multiple resources would be necessary to implement story [156744753](https://www.pivotaltracker.com/n/projects/2129823/stories/156744753) on `JobRuns`, which has a stronger use case.

Reply:
I'd like to propose a different strategy for doing deletion on multiple resources that would look a little more like https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-delete-by-query.html. For now I'd suggest separating out this part of the PR so you can get basic DELETE merged in a more timely fashion 🙇